src/ai/backend/manager/actions/processor.py

```diff
@@ -53,5 +64,12 @@ async def wait_for_complete(self, action: TAction) -> TActionResult:
             await monitor.done(action, result)
         return result.result
 
-    async def fire_and_forget(self, action: TAction) -> None:
-        asyncio.create_task(self.wait_for_complete(action))
+    # TODO: background_task_manager를 ActionProcessor 생성자에서 받아오는 게 나은지?
+    async def fire_and_forget(
+        self, background_task_manager: BackgroundTaskManager, action: TAction
+    ) -> uuid.UUID:
+        async def _bg_task(reporter: ProgressReporter) -> DispatchResult:
+            result = await self.wait_for_complete(action)
+            return result.to_bgtask_result()
+
+        return await background_task_manager.start(_bg_task)
```

Comment:
If you're trying to behave as fire_and_forget, it doesn't seem necessary to get an id.